src/qt/guiutil.h

```diff
@@ -77,6 +77,14 @@ namespace GUIUtil
      */
     QList<QModelIndex> getEntryData(QAbstractItemView *view, int column);
 
+    /** Check whether a field of the currently selected entry of a view is not empty. Does nothing if
+        nothing is selected.
+       @param[in] column  Data column to extract from the model
+       @param[in] role    Data role to extract from the model
+       @see  TransactionView::contextualMenu
+     */
+    bool hasEntryData(QAbstractItemView *view, int column, int role);
```

Comment:
Looks like this should be `const` but `getEntryData` is not. Would that be a big change?